docs/src/test-advanced.md

```diff
@@ -200,6 +201,57 @@ export const test = base.extend<{ saveLogs: void }>({
 });
 ```
 
+## Launching a development web server during the tests
```

Comment:
I think this warrants a separate doc. Maybe combined with "how to build and run tests", or "how to integrate with my dev workflow". Perhaps in a follow up PR.